src/lib/viewers/BaseViewer.js

```diff
@@ -577,14 +578,12 @@ class BaseViewer extends EventEmitter {
      *
      * @protected
      * @param {string} option - to get
-     * @return {Object} Value of a viewer option
+     * @return {Object|undefined} Value of a viewer option
      */
     getViewerOption(option) {
         const { viewers, viewer } = this.options;
-        if (viewers && viewers[viewer.NAME]) {
-            return viewers[viewer.NAME][option];
-        }
-        return null;
```

Comment:
Any chance changing from `null` to `undefined` here will make a difference?

Reply:
I looked through where we use `getViewerOption()` and it doesn't look like there will be a problem.